TESTOWNERS

```diff
@@ -53,6 +55,7 @@
 /dev/devicelab/bin/tasks/picture_cache_perf__e2e_summary.dart @zanderso @flutter/engine
 /dev/devicelab/bin/tasks/platform_channels_benchmarks.dart @gaaclarke @flutter/engine
 /dev/devicelab/bin/tasks/platform_views_scroll_perf__timeline_summary.dart @zanderso @flutter/engine
+/dev/devicelab/bin/tasks/plugin_dependencies_test.dart @stuartmorgan @flutter/plugin
```

Comment:
This one is also `@flutter/tool`.  I can take this one.

Reply:
Updated. Thanks.